.fossa.yaml

```diff
@@ -16,7 +16,7 @@ analyze:
   #     type: folder
   modules:
     - name: fossa-cli
-      path: github.com/fossas/fossa-cli/cmd/fossa
+      path: ./cmd/fossa
```

Comment:
awesome. is this backwards compatible?  we should be OK w/ breaking compatibility until 1.0